### PR TITLE
setting machine fqdn as env variable

### PR DESCRIPTION
**Pull request by geoaxis**
this can help in easy creation of an AWS opsworks rails app.

### CODE REVIEW

File: deploy/before_migrate.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+rails_env = new_resource.environment["RAILS_ENV"]
+secret_key = new_resource.environment["SECRET_KEY_BASE"]
+Chef::Log.info("Precompiling assets for RAILS_ENV=#{rails_env}...")
```
Comment (flavio) [inline deploy/before_migrate.rb]: This is going to fail if Chef is not in place. Can't we simply use Rails' logger?

**Comment by flavio** [comment]
Can you also fix Rubocop's issue? It's just a matter of running `rubocop -a`

**Comment by jordimassaguerpla** [comment]
I just added the database configuration to the appliance as an overlay file, so that we could merge this once the issues flavio mentions are fixed:

https://build.opensuse.org/package/rdiff/Virtualization:containers:Portus/PortusAppliance?linkrev=base&rev=55

File: deploy/before_migrate.rb (hunk 1 above)
Comment (flavio) [inline deploy/before_migrate.rb]: I just realized this file is considered only by Chef at deployment time. So ignore my comment.

**Comment by flavio** [comment]
I'm going to take care of rubocop. Thanks for your submission.

**Pull request merged**